From 137d4d0cb1b37c12dab543c9c5ff23c44a7d7292 Mon Sep 17 00:00:00 2001
From: Dominik Poggel <pog@iesy.com>
Date: Thu, 8 Feb 2024 10:00:07 +0100
Subject: [PATCH 33/36] arm64: dts: iesy: update Makefile with renamed dts

The dts files for the px30 evaluation kits are now named according to their hardware revision. Update the Makefile to include this changes

Signed-off-by: Dominik Poggel <pog@iesy.com>
---
 arch/arm64/boot/dts/iesy/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/iesy/Makefile b/arch/arm64/boot/dts/iesy/Makefile
index c0121a444f7c..ab99acaef40e 100644
--- a/arch/arm64/boot/dts/iesy/Makefile
+++ b/arch/arm64/boot/dts/iesy/Makefile
@@ -1,2 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0
-dtb-$(CONFIG_ARCH_ROCKCHIP) += iesy-rpx30-eva-mi.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += iesy-rpx30-eva-mi-v1.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += iesy-rpx30-eva-mi-v2.dtb
-- 
2.30.2

